src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy

```diff
@@ -121,7 +121,7 @@ class NUnit extends ConventionTask {
 
     String getNunitName() {
         if (isV35OrAbove) {
-            return "NUnit.${getNunitVersion()}"
+            return "NUnit.Console-${getNunitVersion()}"
```

Comment:
They're trolling us, they put it back in the meantime :trollface: (it wasn't there before, I checked)
Good to go with this one ?